lib/marc/record.rb

```diff
@@ -1,7 +1,9 @@
-module MARC  
-    
+require 'forwardable'
+
+module MARC
+
   # The FieldMap is an Array of DataFields and Controlfields.
-  # It also contains a Hash representation 
+  # It also contains a Hash representation
   # of the fields for faster lookups (under certain conditions)
   class FieldMap < Array
```

Comment:
No dependencies on ActiveSupport in ruby-marc. 

But I think hashes in standard ruby are always ordered from ruby 1.9+, if that's all you need.